tests/it/main.rs

```diff
@@ -5,6 +5,7 @@ const STREAM_DATA: &[u8] = include_bytes!("fixtures/fcos-stream.json");
 
 #[test]
 fn test_basic() {
+    println!("hello");
```

Comment:
Why is this here?

Reply:
I ll remove it.  I was trying out the print statements. Will remove it.